packages/apps/plugins/plugin-deck/src/types.ts

```diff
@@ -16,12 +16,16 @@ import type {
 export const NewPlankPositions = ['start', 'end'] as const;
 export type NewPlankPositioning = (typeof NewPlankPositions)[number];
 
+export const OverScrollOptions = ['none', 'centering'] as const;
+export type OverScroll = (typeof OverScrollOptions)[number];
```

Comment:
Teeny nit, I think you rightly treat this as a tightly-compounded word elsewhere, but loosely-compounded here — maybe:
```suggestion
export const OverscrollOptions = ['none', 'centering'] as const;
export type Overscroll = (typeof OverscrollOptions)[number];
```